Assets/HotmaxEditorVR/gizmoScaler.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class gizmoScaler : MonoBehaviour
+{
+    public float sizeScaler;
+
+	void Update ()
+    {
+        float distanceToHead = Vector3.Distance(transform.position, init.vrCamera.transform.position) / sizeScaler;
```

Comment:
variable should include the `/ sizeScaler` --> make another variable that is `scaleRatio = distanceToHead / sizeScaler`